test/lib/ufe/testDeleteCmd_v2.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python
+
+#
+# Copyright 2020 Autodesk
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import maya.cmds as cmds
+
+from ufeTestUtils import usdUtils, mayaUtils
+import ufe
+
+import unittest
+
+import os
+
+def childrenNames(children):
+    return [str(child.path().back()) for child in children]
+
+class TestObserver(ufe.Observer):
+    def __init__(self):
+        super(TestObserver, self).__init__()
+        self.deleteNotif = 0
+        self.addNotif = 0
+
+    def __call__(self, notification):
+        if isinstance(notification, ufe.ObjectDelete):
+            self.deleteNotif += 1
+        if isinstance(notification, ufe.ObjectAdd):
+            self.addNotif += 1
+
+    def nbDeleteNotif(self):
+        return self.deleteNotif
+
+    def nbAddNotif(self):
+        return self.addNotif
+
+    def reset(self):
+        self.addNotif = 0
+        self.deleteNotif = 0
+
+class DeleteCmdTestCase(unittest.TestCase):
+    '''Verify the Maya delete command, for multiple runtimes.
+
+    UFE Feature : SceneItemOps
+    Maya Feature : delete
+    Action : Remove object from scene.
+    Applied On Command Arguments:
```

Comment:
This entire file is a copy of the original with just this one test. I changed this comment to be from command args (instead of selection).